### ISSUE TITLE
laravel 5.5 new pagination API

### ISSUE THREAD

**Issue by flyingcoder** [ratiw/vuetable-2#307]
With laravel 5.5 the response json is returning items instead of data.

Is there a way I can tell vuetable-2 to listen to items instead of data?

`LengthAwarePaginator {#831 ▼
  #total: 103
  #lastPage: 21
  #items: Collection {#863 ▼
    #items: array:5 [▶]
  }
  #perPage: 5
  #currentPage: 1
  #path: "http://buzzcrm.vin/services"
  #query: []
  #fragment: null
  #pageName: "page"
}`

**Comment by ratiw** [comment]
According to [Laravel doc](https://laravel.com/docs/5.5/pagination#converting-results-to-json), the default is still `data`, but if you would like Vuetable to look at something else, use the [`data-path` prop](https://ratiw.github.io/vuetable-2/#/Vuetable-Properties?id=-data-path)

**Comment by Dimidel** [comment]
Pagination works with the setting : pagination-path="meta"
Hope it helps

**Comment by flyingcoder** [comment]
@ratiw yes the docs I think is not updated. Anyway thank you I will try `data-path` prop
